src/httpserver.cpp

```diff
@@ -269,7 +269,7 @@ static void http_request_cb(struct evhttp_request* req, void* arg)
             item.release(); /* if true, queue took ownership */
         else {
             LogPrintf("WARNING: request rejected because http work queue depth exceeded, it can be increased with the -rpcworkqueue= setting\n");
-            item->req->WriteReply(HTTP_INTERNAL_SERVER_ERROR, "Work queue depth exceeded");
+            item->req->WriteReply(HTTP_SERVICE_UNAVAILABLE, "Work queue depth exceeded");
```

Comment:
Does this need release notes to explain the changed http status code? Just in case a user relies on this?

Just asking, you don't have to invalidate review. Release note can be added after merge.

Reply:
Good point, I think you're right, I'll do it separately, thanks.

Reply:
You can add them directly to `doc/release-notes.md` on the `master` branch

Reply:
Maybe add a "needs release note" label.